test/broadcast.jl

```diff
@@ -300,7 +300,6 @@ import Base.Meta: isexpr
 # PR 16988
 @test Base.promote_op(+, Bool) === Int
 @test isa(broadcast(+, [true]), Array{Int,1})
-@test Base.promote_op(Float64, Bool) === Float64
```

Comment:
Are you dropping this test because it would fail now? That would be unfortunate in case something outside Base relies on it...